support/jakarta-ee/src/main/java/org/apache/shiro/ee/cdi/ShiroSecurityInterceptor.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.shiro.ee.cdi;
+
+import org.apache.shiro.ee.cdi.AopHelper.SecurityInterceptor;
+import java.io.Serializable;
+import java.util.List;
+import javax.annotation.Priority;
+import javax.enterprise.context.Dependent;
+import javax.interceptor.AroundInvoke;
+import javax.interceptor.Interceptor;
+import javax.interceptor.InvocationContext;
+
+/**
+ * Enforce Shiro security on EJBs and CDI Beans
+ *
+ * <a href="http://code.google.com/p/flowlogix/wiki/TLShiroSecurityInterceptor"
```

Comment:
Broken link? Documentation might need to be added to shiro?

Reply:
yup, old link